### PR TITLE
Update stage 0 and remove Target Platform work around

### PR DESCRIPTION
**Pull request by sfoslund**

### CODE REVIEW

File: src/Assets/Directory.Build.props [hunk 1]
```diff
@@ -3,4 +3,8 @@ Sometimes it is useful to play around with the test assets
 directly where you're authoring them, and they should never
 get the build customization from the rest of the repo. 
 -->
-<Project />
\ No newline at end of file
+<Project>
+  <PropertyGroup>
+    <CheckEolTargetFramework>false</CheckEolTargetFramework>
```
Comment (wli3) [inline src/Assets/Directory.Build.props:8]: nice!

**Review by wli3 — APPROVED** [review]

**Comment by sfoslund** [comment]
We need to get an installer build with the changes in https://github.com/dotnet/sdk/pull/13093 to get tests passing here.

**Comment by sfoslund** [comment]
@swaroop-sridhar some crossgen tests are failing here after updating stage 0, do you know what might be going on?

It_collects_crossgen2_publishing_properties
> F:\workspace\_work\1\s\artifacts\bin\redist\Release\dotnet\sdk\5.0.100-ci\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Publish.targets(394,5): error : Error: [F:\workspace\_work\1\s\artifacts\tmp\Release\It_collects_c---0354E0EC\TrimmedR2RSingleFileProject\TrimmedR2RSingleFileProject.csproj]
F:\workspace\_work\1\s\artifacts\bin\redist\Release\dotnet\sdk\5.0.100-ci\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Publish.targets(394,5): error :   An assembly specified in the application dependencies manifest (crossgen2.deps.json) was not found: [F:\workspace\_work\1\s\artifacts\tmp\Release\It_collects_c---0354E0EC\TrimmedR2RSingleFileProject\TrimmedR2RSingleFileProject.csproj]
F:\workspace\_work\1\s\artifacts\bin\redist\Release\dotnet\sdk\5.0.100-ci\Sdks\Microsoft.NET.Sdk\targets\Microsoft.NET.Publish.targets(394,5): error :     package: 'runtimepack.Microsoft.NETCore.App.Runtime.win-x64', version: '5.0.0-preview.8.20361.2'

**Comment by swaroop-sridhar** [comment]
CC: @AntonLapounov 
Is the crossgen package missing api-ms-win-core-console-l1-2-0.dll?

**Comment by sfoslund** [comment]
@swaroop-sridhar it looks like in the past these tests have been disabled while the issue is being fixed on your end, does that make sense to do here too?

**Comment by swaroop-sridhar** [comment]
> @swaroop-sridhar it looks like in the past these tests have been disabled while the issue is being fixed on your end, does that make sense to do here too?

Sounds reasonable to me. Please create an issue to enable these tests, and assign it to @mangod9's team.

**Comment by sfoslund** [comment]
Great, the issue is here: https://github.com/dotnet/sdk/issues/13279

**Comment by mangod9** [comment]
Is this blocking 5.0?

**Comment by sfoslund** [comment]
> Is this blocking 5.0? 

This needs to go in 5.0, yes. Ideally we'd like to get it into RC1 (today), but RC2 is fine is not. Reenabling the tests is not blocking.

**Comment by AntonLapounov** [comment]
@swaroop-sridhar Yes, we have lost that DLL recently. Looking into that.

**Pull request merged**